sark/sark_io.c

```diff
@@ -409,17 +434,32 @@ void io_put_fixed(char *stream, uint n, uint d, uint a, uint pad, int neg)
 }
 
  // <drl add>
+//! \brief Format for s15.16 fixed point
+//! \details Delegates to io_put_fixed()
+//! \param[in] stream: Where to write to
+//! \param[in] n: signed accum, as signed integer
+//! \param[in] d: Field width
+//! \param[in] a: Precision
+//! \param[in] pad: Padding character
 static void io_put_sfixed(char *stream, int n, uint d, uint a, uint pad)
 {
     io_put_fixed(stream, (uint)((n < 0)? (-n): n) << 1, d, a, pad, (n < 0));
 }
 
+//! \brief Format for 16.16 fixed point
```

Comment:
Is it worth clarifying that this is u16.16 ?

Reply:
Done